input/pagecontent/changes.md

```diff
@@ -93,6 +93,7 @@ To help implementers, only the more significant changes are listed here.
     </ul>
   </li>
   <li>Changed context of extension <a href="StructureDefinition-indigenous-status.html">Australian Indigenous Status</a> to add Person and RelatedPerson.</li>
+  <li>Added AU Pensioner Concession Card Number, AU Commonwealth Seniors Health Card Number and AU Health Care Card Number to Coverage.identifier in <a href="StructureDefinition-au-coverage.html">AU Base Coverage</a> (<a href="https://jira.hl7.org/browse/FHIR-47191">FHIR-47191</a>).</li>
```

Comment:
Suggest change to align wording with similar entries: 
Changed Coverage.identifer type in AU Base Coverage to add AU Pensioner Concession Card Number, AU Commonwealth Seniors Health Card Number and AU Health Care Card Number (<a href="https://jira.hl7.org/browse/FHIR-47191">FHIR-47191</a>).

Reply:
Amended, thanks.

Reply:
Confirmed